package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/checkout-components",
-  "version": "5.0.286",
+  "version": "5.0.288",
```

Was this change meant to be included?
```suggestion
  "version": "5.0.286",
```

---

hmm, no, good catch let me fix that. Must be an artifact of feature-branch alpha versions?